hydra-node/hydra-node.cabal

```diff
@@ -239,6 +238,27 @@ benchmark tx-cost
     -- NOTE(SN): should fix HLS choking on PlutusTx plugin
     ghc-options: -fplugin-opt PlutusTx.Plugin:defer-errors
 
+benchmark micro
+  import:         project-config
+  hs-source-dirs: exe/micro-bench
```

Comment:
We usually keep benchmarks in the `bench/` directory within a package.

Should: use `bench/` as a source directory

Reply:
Fair enough. The other benchmark we have is in the `exe` directory hence I followed suit. Happy to move them both.

Reply:
You're right. `tx-cost` is different than the other 3 benchmarks we have on the repository. I'd say we should move all to `bench/`